subsystems/mvdm/ntvdm/dos/dos32krnl/process.c

```diff
@@ -796,10 +796,40 @@ WORD DosCreateProcess(IN LPCSTR ProgramName,
         /* Those are handled by NTVDM */
         case SCS_WOW_BINARY:
         {
-            DisplayMessage(L"Trying to load '%S'.\n"
-                           L"WOW16 applications are not supported internally by NTVDM at the moment.\n"
-                           L"Press 'OK' to continue.",
-                           ProgramName);
+            CHAR AppName[50] = "\"%programfiles%\\otvdm\\otvdmw.exe\" ";
+            STARTUPINFOA si;
+            CHAR ExpName[256];
+            ExpandEnvironmentStringsA(AppName, ExpName, 255);
+            strcat(ExpName, "\"");         // Add double-quote before ProgramName
+            strcat(ExpName, ProgramName);  // Append Program name
+            strcat(ExpName, "\"");         // Add double-quote after ProgramName
```

Comment:
If strsafe functions are available here, better use them.

Reply:
Though:
![image](https://user-images.githubusercontent.com/33393466/142517865-ed89a211-2643-4782-9853-27acd6b1d19a.png)

Reply:
I did a search similar to @JoachimHenze and was not able to find any reference to 'strsafe' in the 'subsystems' folder. Therefore, I am leaving these as they are for now. Thanks.